docs/reference/pip_install.rst

```diff
@@ -89,6 +93,25 @@ Some Examples:
   Don't use single or double quotes in a ``requirements.txt`` file.
 
 
+.. _`Per-requirement Overrides`:
+
+Per-requirement Overrides
++++++++++++++++++++++++++
+
+It is possible to set ``--install-options`` and ``--global-options``
+for each requirement in the requirements file:
+
+ ::
+
+    FooProject >= 1.2 --install-options="--prefix='/usr/local'" \
+                      --global-options="--no-user-cfg"
+
+The above translates roughly into running FooProject's ``setup.py``
+script as:
+
+ ::
+
+    python setup.py --no-user-cfg install --prefix='/usr/local'
```

Comment:
looks like the behavior of the isolated option (which adds `--no-user-cfg`)  has changed somehow with this.  not sure where atm w/o looking further.